api/v1/infinispan_webhook_test.go

```diff
@@ -55,34 +55,6 @@ var _ = Describe("Infinispan Webhooks", func() {
 	})
 
 	Context("Infinispan", func() {
-		It("Should initiate Cache Service defaults", func() {
```

Comment:
We should add a new test which ensures that `FieldValueForbidden` is returned when a user tries to create a CR with the CacheService type.